README.md

```diff
@@ -115,21 +115,19 @@ Hit us up if you're interested in using it or want to collaborate with us. [We'r
 
 ## Install
 
-First, [install Docker if you haven't already](https://docs.docker.com/get-docker/). Then, run this in a terminal:
+You can download and install the latest release of Cog
+by running the following commands in a terminal:
 
-```
-sudo curl -o /usr/local/bin/cog -L https://github.com/replicate/cog/releases/latest/download/cog_`uname -s`_`uname -m`
+```console
+sudo curl -o /usr/local/bin/cog -L "https://github.com/replicate/cog/releases/latest/download/cog_$(uname -s)_$(uname -m)"
 sudo chmod +x /usr/local/bin/cog
 ```
 
-## Upgrade
```

Comment:
I'm sure if there are many links to `#upgrade` out in the wild, but maybe we could have an invisible anchor here just in case?